src/_components/guides/author.rb

```diff
@@ -0,0 +1,5 @@
+class Guides::Author < Bridgetown::Component
+  def initialize(author:)
+    @author = Bridgetown::Current.site.data.authors[author]
```

Comment:
Here @author can be nil. You have to add a safe guard either here (def render false ) or in your html with blank data.